config.py

```diff
@@ -965,12 +976,39 @@ def _update_python_path(check_config):
         sys.path.extend(pythonpath)
 
 
+def validate_sdk_check(manifest_path):
+    max_validated = min_validated = False
+    try:
+        with open(manifest_path, 'r') as fp:
+            manifest = json.load(fp)
+            for maxfield in MANIFEST_VALIDATION['max']:
+                max_version = manifest.get(maxfield)
+                if not max_version:
+                    continue
+
+                max_validated = False if max_version < get_version() else True
+                break
+
+            for minfield in MANIFEST_VALIDATION['min']:
+                min_version = manifest.get(minfield)
+                if not min_version:
+                    continue
+
+                min_validated = False if min_version > get_version() else True
+                break
+    except IOError:
+        log.warn("Manifest file (%s) not present " % manifest_path)
```

Comment:
I think it's unnecessary to get two warnings in a row. Can you add this to the warning below instead of here? 

This might be valuable as a debug log line, but I don't think you need it as a warning.

Is an IOError the only error you can get here? I think you will also want to catch json parsing errors.